Symfony/CS/Tests/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixerTest.php

```diff
@@ -212,4 +212,26 @@ public function testFixesWindowsStyle()
 
         $this->makeTest($expected, $input);
     }
+
+    // empty line between typehinting docs and return statement should be preserved
+    public function testInlineTypehintingDocsBeforeReturn()
+    {
+        $expected = <<<'EOF'
+<?php
+
+function parseTag($tag)
+{
+    $tagClass = get_class($tag);
+
+    if ('phpDocumentor\Reflection\DocBlock\Tag\VarTag' === $tagClass) {
+        /** @var DocBlock\Tag\VarTag $tag */
```

Comment:
Indeed it does not document anything here, so we could indeed avoid the removal. On the other hand, those using it wrong would still expect to have removal of the blankline, while people using docblocks correctly would never run into this problem.
About priority, doc_to_comment already runs before all other docblock related fixers, so we are fine there.
I guess removing the blankline is a matter of interpretation here. Let me know your final verdict on this :)